apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/ListBucketsTest.cs

```diff
@@ -75,6 +75,38 @@ public void PartialResponses()
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void ListBuckets_ReturnPartialSuccess(bool returnPartialSuccess)
```

Comment:
I don't see the point of having this integration test. We really cannot test anything, and even if we could it would be server behaviour that we usally don't test client side.